freqtrade/constants.py

```diff
@@ -470,6 +471,15 @@
         },
         'position_adjustment_enable': {'type': 'boolean'},
         'max_entry_position_adjustment': {'type': ['integer', 'number'], 'minimum': -1},
+        'orderflow': {
+            'type': 'object',
+            'properties': {
+                'scale': {'type': 'number', 'minimum': 0.0},
+                'stacked_imbalance_range': {'type': 'number'},
+                'imbalance_volume': {'type': 'number'},
+                'imbalance_ratio': {'type': 'number'},
```

Comment:
@TheJoeSchr I'd apreciate if you could have a look at these 
i assume we're able to define ranges for some of these ... though from the docs, the actual range wasn't clear to me - and i (intentionally) didn't want to discover it by reading the code.

Reply:
> Also, i ASSUME most are mandatory .. though i'm not entirely sur, so i'd rather leave that to you.

How should I make them mandatory? Via `freqtrade/configuration/config_validation.py`: `_validate_orderflow`?

Reply:
e.g. like this: `'required': ['scale']`

json-schema supports this - check for "required" in this file on usage of this ;)